packages/marko-web-identity-x/browser/authenticate.vue

```diff
@@ -247,6 +247,11 @@ export default {
           isProfileComplete: this.isProfileComplete,
           requiresCustomFieldAnswers: this.requiresCustomFieldAnswers,
           actionSource: data.loginSource,
+          ...(
+            data.additionalEventData
+            && data.additionalEventData.newsletterSignupType
+            && { newsletterSignupType: data.additionalEventData.newsletterSignupType }
+          ),
```

Comment:
Remove this, this will be present under `additionalEventData.newsletterSignupType` below

Reply:
@solocommand This is how P1 Events expects it to be sent in order to send it accordingly via this: https://github.com/parameter1/base-cms/blob/master/packages/marko-web-p1-events/browser/index.js#L69